pkgs/build-support/rust/default-crate-overrides.nix

```
@@ -20,6 +20,7 @@
   gmp,
   gobject-introspection,
   graphene,
+  gst_all_1,
```

```suggestion
  gst_all_1,
  gst_plugins,
```
Since you use it below